kibbeh/src/modules/keyboard-shortcuts/ChatKeybind.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from "react";
 import { recordKeyCombination } from "react-hotkeys";
 import { useKeyMapStore } from "../../global-stores/useKeyMapStore";
 import { Button } from "../../ui/Button";
+import {VisualKeybind} from "./VisualKeybind";
```

Comment:
try to keep formatting consistent with the rest of the code - `import { VisualKeybind } ...`